.github/workflows/ci.yml

```
@@ -0,0 +1,27 @@
+name: Unit test
+on: [pull_request]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Read .nvmrc
+        run: echo NVMRC=$(cat .nvmrc) >> $GITHUB_OUTPUT
+        id: nvm
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '${{ steps.nvm.outputs.NVMRC }}'
+      - name: Install dependencies
+        run: npm ci
+      - name: Run unit tests and collect coverage
+        run: npm run test
```

We already have a test script defined for coverage. Please use the same.
```suggestion
        run: npm run test:coverage
```